__tests__/app.test.js

```diff
@@ -245,3 +245,20 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test.only("should add a comment to given article ID", () => {
+    const newComment = {
+      username: "icellusedkars",
+      comment: "Is this even going to work?",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .expect(201)
+      .send(newComment)
+      .then(({ body }) => {
+        expect(body.comment.body).toEqual(newComment.comment);
+        expect(body.comment.author).toBe(newComment.username);
+      });
+  });
+});
```

Comment:
we need to add tests for the following situations:
404-review not found
400-invalid review id
404-username not found (we only want registered users posting)
400-missing required field (body etc)